src/lifecycle/index.tsx

```diff
@@ -226,6 +253,7 @@ function reducer(previousState: State, action: Action): State {
           connectivity: { type: 'Following', endpoints },
           block: previousState.block,
           chain: previousState.chain,
```

Comment:
Also here if chain is set to previousState.chain in updates, does this mean it will not pull the changes in votes and refrendas?!